Bug 1112596 - Fix loading indicator not showing on load. r=alive

Cwiiis opened the pull request:
If a page triggers a loadstart before the app chrome loads, the app chrome
will not show the loading state.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/668518?v=3" alt="Cwiiis" width="50px" height="50px" /> Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fcaa6c040bfe)

Cwiiis committed:
Bug 1112596 - Fix loading indicator not showing on load. r=alive

If a page triggers a loadstart before the app chrome loads, the app chrome
will not show the loading state.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/668518?v=3" alt="Cwiiis" width="50px" height="50px" /> Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bc4279dffc01)

alivedise commented on apps/system/js/app_window.js:
```diff
@@ -792,12 +792,15 @@
+          if (that.inError) {
             that.appChrome.handleEvent({type: 'mozbrowsererror'});
           }
+          if (that.loading) {
```
Let's test this in app_window_test

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/668518?v=3" alt="Cwiiis" width="50px" height="50px" /> Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7f6728ffef68)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/668518?v=3" alt="Cwiiis" width="50px" height="50px" /> Chris Lord (Cwiiis)](https://github.com/Cwiiis) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95ff5ef8a6e9)
